src/abstractdataframe/reshape.jl

```diff
@@ -413,99 +412,61 @@ function CategoricalArrays.CategoricalArray(v::RepeatedVector)
     res
 end
 
-"""
-    stackdf(df::AbstractDataFrame, [measure_vars], [id_vars];
-            variable_name::Symbol=:variable, value_name::Symbol=:value)
-    meltdf(df::AbstractDataFrame, [id_vars], [measure_vars];
-           variable_name::Symbol=:variable, value_name::Symbol=:value)
-
-Return a stacked view of a data frame (long format).
-
-Like [`stack`](@ref) and [`melt`](@ref), but a view is returned rather than data
-copies.
-
-Return a `DataFrame` with a column `value_name` (`:value` by default)
-holding the values of the stacked columns (`measure_vars`), with
-column `variable_name` (`:variable` by default) a vector of `Symbol`s with the `measure_vars` name,
-and with columns for each of the `id_vars`.
-
-The result is a view because the columns are special `AbstractVectors`
-that return indexed views into the original data frame.
-
-# Arguments
-- `df` : the wide AbstractDataFrame
-- `measure_vars` : the columns to be stacked (the measurement
-  variables), a normal column indexing type, like a `Symbol`,
-  `Vector{Symbol}`, Int, etc.; for `meltdf`, defaults to all
-  variables that are not `id_vars`
-- `id_vars` : the identifier columns that are repeated during
-  stacking, a normal column indexing type; for `stackdf` defaults to all
-  variables that are not `measure_vars`
-
-# Examples
-```julia
-d1 = DataFrame(a = repeat([1:3;], inner = [4]),
-               b = repeat([1:4;], inner = [3]),
-               c = randn(12),
-               d = randn(12),
-               e = map(string, 'a':'l'))
-
-d1s = stackdf(d1, [:c, :d])
-d1s2 = stackdf(d1, [:c, :d], [:a])
-d1m = meltdf(d1, [:a, :b, :e])
-```
-"""
-(stackdf, meltdf)
-
-function stackdf(df::AbstractDataFrame, measure_vars::AbstractVector{<:Integer},
+function _stackdf(df::AbstractDataFrame, measure_vars::AbstractVector{<:Integer},
                  id_vars::AbstractVector{<:Integer}; variable_name::Symbol=:variable,
```

Comment:
Wrong indentation here and below.

But maybe better rename the internal function while you're at it? E.g. `stackview` or `viewstack`?

Reply:
ok. fixed